backend/src/main/scala/cromwell/backend/validation/ValidatedRuntimeAttributesBuilder.scala

```diff
@@ -47,25 +47,13 @@ trait ValidatedRuntimeAttributesBuilder {
 
   def unsupportedKeys(keys: Seq[String]): Seq[String] = keys.diff(validationKeys)
 
-  private lazy val coercionMap: Map[String, Traversable[WdlType]] =
-    validations.map(attr => attr.key -> attr.coercion).toMap
-
   private lazy val validationKeys = validations.map(_.key)
 
-  private lazy val staticDefaults: Map[String, WdlValue] = (validations collect {
-    case attr if attr.staticDefaultOption.isDefined => attr.key -> attr.staticDefaultOption.get
-  }).toMap
-
-  def build(attrs: Map[String, WdlValue], options: WorkflowOptions, logger: Logger): ValidatedRuntimeAttributes = {
-    import RuntimeAttributesDefault._
-
+  def build(attrs: Map[String, WdlValue], logger: Logger): ValidatedRuntimeAttributes = {
     // Fail now if some workflow options are specified but can't be parsed correctly
-    val defaultFromOptions = workflowOptionsDefault(options, coercionMap).get
-    val withDefaultValues: Map[String, WdlValue] = withDefaults(attrs, List(defaultFromOptions, staticDefaults))
-
-    RuntimeAttributesValidation.warnUnrecognized(withDefaultValues.keySet, validationKeys.toSet, logger)
+    RuntimeAttributesValidation.warnUnrecognized(attrs.keySet, validationKeys.toSet, logger)
 
-    val runtimeAttributesErrorOr: ErrorOr[ValidatedRuntimeAttributes] = validate(withDefaultValues)
+    val runtimeAttributesErrorOr: ErrorOr[ValidatedRuntimeAttributes] = validate(attrs)
```

Comment:
Happy to discuss more of what was there, but in general the `attrs` will have to undergo coercion before being passed to `validate()`.

I copy/pasted this functionality of `workflowOptionsDefault` from elsewhere. I assume based on its removal that it was doing a bit too much, perhaps also injecting defaults into the attributes?

However, without usage of the `coercionMap` at all, the SFS docker specs are currently failing because the `Map("docker" -> WdlExpression(…))` is not being coerced into a `Map("docker" -> WdlString(…))` before validation.